extensions-contrib/kubernetes-overlord-extensions/src/main/java/org/apache/druid/k8s/overlord/common/KubernetesPeonClient.java

```diff
@@ -56,10 +56,10 @@ public Pod launchPeonJobAndWaitForStart(Job job, long howLong, TimeUnit timeUnit
     // launch job
     return clientApi.executeRequest(client -> {
       client.batch().v1().jobs().inNamespace(namespace).resource(job).create();
-      K8sTaskId taskId = new K8sTaskId(job.getMetadata().getName());
-      log.info("Successfully submitted job: %s ... waiting for job to launch", taskId);
+      String jobName = job.getMetadata().getName();
+      log.info("Successfully submitted job: %s ... waiting for job to launch", jobName);
```

Comment:
It would be good to report the taskId in this log line, to make it easy to map to the task_id that is being started.